docs/api/v1/resourcesetinputprovider.md

```diff
@@ -184,6 +184,38 @@ spec:
     name: github-pat
 ```
 
+#### GitHub App authentication
+
+For GitHub, GitHub App authentication is also supported. Instead of adding the basic
+auth keys `username` and `password`, you can add the following GitHub App keys to the
+secret:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: github-app
+  namespace: default
+stringData:
+  githubAppID: "<GITHUB APP ID>"
+  githubAppInstallationID: "<GITHUB APP INSTALLATION ID>"
+  githubAppPrivateKey: |
+    -----BEGIN RSA PRIVATE KEY-----
+    ...
+    -----END RSA PRIVATE KEY-----
```

Comment:
We also support Github Enterprise, please add:
```
githubAppBaseURL: "<github-enterprise-api-url>" #optional for GitHub Enterprise self-hosted
```

Reply:
fixed, thanks